emacs/init.sls

```diff
@@ -1,2 +1,108 @@
-emacs:
-  pkg.installed
+{% from 'emacs/settings.sls' import emacs with context %}
+
+{%- if emacs.from_pkg == True %}
+## This is the default behavior, milk-toast if you will.
+boring-emacs:
+  pkg.installed:
+    - name: emacs
+
+{% else %}
+# You have chosen wisely but your journey to the great Basillica of St. Stallman has only begin!
+{% set build_dir = '/usr/local/src' %}
```

Comment:
Installing from package is the default behavior 
https://github.com/jackscott/emacs-formula/blob/2f3c9b93880eae98179d57b58d5a1690557b5611/emacs/settings.sls#L8